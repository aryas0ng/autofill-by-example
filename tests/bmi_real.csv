height,weight,bmi
165,49.5,18.2
156,56.25,23.1
169,55.79,19.5
174,69.2,22.9
167,55.79,
180,70.42,
164,68.49,
154,77.56,
171,54.2,
151.5,83.55,
160,53.5,
162,61.8,
156,49.8,
169,59.8,
166,88.9,
175,80.73,
166,73.9,
161,56.7,
170,70.3,
164,66.9,
172,73.48,
152,51.94,
166.5,60.5,
165,63.73,
163,78.9,
164,50.08,
161,68,
171.5,91,
170,63.5,
163.5,60.96,
161,66.2,
164,80.78,
171.5,55.8,
162.5,64,
175.5,57,
156.5,50.3,
162,67.49,
171,67.1,
,,
165,68.9,
171,58.9,
156,65.8,
154,51.3,
171.5,70.8,
166,73.5,
164,89.8,
163,78.6,
165,56.7,
163,66.7,
165,93.44,
173,73.9356,
154,57.15,
,,
166,64.18,
160,61.69,
155,98.88,