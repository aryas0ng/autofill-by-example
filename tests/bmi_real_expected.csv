height,weight,bmi
165,49.5,18.2
156,56.25,23.1
169,55.79,19.5
174,69.2,22.9
167,55.79,20
180,70.42,21.7
164,68.49,25.5
154,77.56,32.7
171,54.2,18.5
151.5,83.55,36.4
160,53.5,20.9
162,61.8,23.5
156,49.8,20.5
169,59.8,20.9
166,88.9,32.3
175,80.73,26.4
166,73.9,26.8
161,56.7,21.9
170,70.3,24.3
164,66.9,24.9
172,73.48,24.8
152,51.94,22.5
166.5,60.5,21.8
165,63.73,23.4
163,78.9,29.7
164,50.08,18.6
161,68,26.2
171.5,91,30.9
170,63.5,22
163.5,60.96,22.8
161,66.2,25.5
164,80.78,30
171.5,55.8,19
162.5,64,24.2
175.5,57,18.5
156.5,50.3,20.5
162,67.49,25.7
171,67.1,22.9
165,68.9,25.3
171,58.9,20.1
156,65.8,27
154,51.3,21.6
171.5,70.8,24.1
166,73.5,26.7
164,89.8,33.4
163,78.6,29.6
165,56.7,20.8
163,66.7,25.1
165,93.44,34.3
173,73.9356,24.7
154,57.15,24.1
166,64.18,23.3
160,61.69,24.1
155,98.88,41.2